Source/WebCore/rendering/RenderAttachment.cpp

```diff
@@ -97,8 +97,18 @@ void RenderAttachment::layout()
         layoutShadowContent(newIntrinsicSize);
 }
 
-LayoutUnit RenderAttachment::baselinePosition(FontBaseline, bool, LineDirectionMode, LinePositionMode) const
+LayoutUnit RenderAttachment::baselinePosition(FontBaseline fontBaseline, bool firstLine, LineDirectionMode lineDirectionMode, LinePositionMode linePositionMode) const
 {
+    if (auto* baselineElement = attachmentElement().wideLayoutElementForBaseline()) {
+        bool unusedIsReplaced;
+        auto attachmentRect = attachmentElement().pixelSnappedRenderRect(&unusedIsReplaced);
+        auto baselineElementRect = baselineElement->pixelSnappedRenderRect(&unusedIsReplaced);
```

Comment:
any particular reason for using snapped geometry here?

Reply:
I copied other code blindly, I guess it's not needed here.